lib/idl/cherami.thrift

```diff
@@ -95,7 +95,13 @@ enum DestinationType {
    * Each message has associated sequence number. Sequence numbers are guaranteed to be sequential.
    * Messages with incorrect sequence number provided by publisher are rejected.
   **/
-  LOG
+  LOG,
```

Comment:
unrelated to this diff but is it possible to use cherami-thrift's idl definition here instead of making a hard copy? May be a git submodule of some sort?